# Course content outline

## UNIT1 1: Getting started

### Week 1: Hello Computational Biology!
Reading: NIH Working definition of bioinformatics and computational biology
Supplemental reading: Markowetz (2017).  All biology is computational biology
Supplemental reading: Poisot et al (2019).  Data based, synthesis drive: Setting the agenda for computational ecology

Supplemental reading: Wikipedia.  Computational biology
 
### Week 2: R Camp
Reading:  Thieme (2018).  R generation
 
### Week 3: Stats Camp
Reading: Meillere et al (2015).  Traffic noise exposure affects telomere length in nestling house sparrows
 
## UNTI 2: Computational Ecology
Reading:  Dennis et al (1991) Estimation of growth and extinction parameters for endangered species
 
See Abstract, Introduction and Grizzly bear case study (pg 131-133).
Reference: Morris and Doak (2002). Quantitative conservation biology.
Reference: Steven (2009).  A primer of ecology in R.
 

## UNIT 3: Phylogenetics & Bioinformatics
Higgs and Atwood 2005.  Bioinformatics and Molecular Evolution
Lycett et al 2007.  Phylogenetic analyses of behavior support existence of culture among wild 
chimpanzees
Lycett et al 2010.  Are behavioral differences among wild chimpanzee communities genetic or 
cultural? An assessment using tool‚Äêuse data and phylogenetic methods.

 
## UNIT 4: Bioinformatics 2 - Sequence Alignment and BLAST searches

Hildebrand and Soriano (1999). Shroom, a PDZ domain-containing actin-binding protein, is 
required for neural tube morphogenesis in mice.  Cell.  (link ; pdf). See section " shrm 
Encodes a Novel PDZ Domain Protein" and Figure 4, pg 488-489.
Hagens, Hildebrand et al (2006). A new standard nomenclature for proteins related to Apx and Shroom.

## Unit 5: Bioinformatics 3 - Algorithms and statistics of BLAST

